api_tests/debug_tools/eth_transaction_trace.js

```diff
@@ -0,0 +1,48 @@
+const ethutil = require("ethereumjs-util");
+const Web3 = require("web3");
+const Api = require("@parity/api");
+
+const provider = new Api.Provider.Http('http://localhost:33056');
```

Comment:
in order to make this file a bit more useable you could allow passing the URL via argument. I'm not 100% but I think it was simple: `process.argv[0]` and then you could start it with: `node eth_transaction_trace.js 127.0.0.1`

Reply:
This has been moved to a gist.